handler.js

```diff
@@ -2,7 +2,7 @@
 function runTest(event, context, cb){ 
 
 	const exec = require('child_process').exec;
-	exec("cp -a FitNesseRoot /tmp; chmod 755 /tmp; java -jar fitnesse-standalone.jar -d '/tmp' -c '" + event.test + "?test&format=xml&includehtml' -b '/tmp/" + event.test + ".xml' -p 8002; cat '/tmp/" + event.test + ".xml'", (error, stdout, stderr) => {
+    exec("export API_GATEWAY=" + event.apy_gateway + "; cp -a FitNesseRoot /tmp; chmod 755 /tmp; java -jar fitnesse-standalone.jar -d '/tmp' -c '" + event.test + "?test&format=xml&includehtml' -b '/tmp/" + event.test + ".xml' -p 8003; cat '/tmp/" + event.test + ".xml'", (error, stdout, stderr) => {
```

Comment:
1. `API_GATEWAY` can be set in more clean way.
2. Looks ugly when you do a command sequence in exec. It's possible to describe `chmod` and `cp` with `nodejs fs.*` api.
3. Is the port  important here? Seems like - not...